src/data/faq_de.json

```diff
@@ -376,6 +379,216 @@
+                            },
+                            {
+                                "title": "Ich war bei einer Veranstaltung per QR-Code eingecheckt. Kann ich die Gäste des Events mit der Funktion warnen? Bleibe ich dabei anonym?",
+                                "anchor": "warn_without_tan_event",
+                                "active": false,
+                                "textblock": [
+                                    "Ja, wenn Sie sich per Event-QR-Code eingecheckt haben, können Sie in dem Zeitraum Ihres Check-ins die anderen Gäste weiterhin warnen. Dabei bleibt, wie bisher, Ihre Identität anonym."
+                                ]
+                            },
+                            {
+                                "title": "Muss ich nach einem positiven (Selbst-)Test in Isolation?",
+                                "anchor": "warn_without_tan_isolation",
+                                "active": false,
+                                "textblock": [
+                                    "Nach jedem positiven Test empfehlen wir im Sinne der Eigenverantwortung, zunächst Kontakte weitgehend zu minimieren und mit dem Arzt Ihres Vertrauens oder dem Gesundheitsamt Kontakt aufzunehmen und die weiteren Schritte abzustimmen.",
+                                    "Die aktuellen Isolations-Regeln richten sich nach dem IfSG und nach den lokalen geltenden Bestimmungen."
+                                ]
+                            },
+                            {
+                                "title": "Wenn ich wieder einen negativen Schnelltest habe, kann ich die Isolation dann beenden?",
+                                "anchor": "warn_without_tan_negative",
+                                "active": false,
+                                "textblock": [
+                                    "Die Corona-Warn-App kann keine medizinische Diagnose stellen, ob Sie infiziert sind oder nicht. Daher klären Sie bitte die Frage der Beendigung der Isolation mit dem zuständigen Gesundheitsamt bzw. Ihrem Arzt."
+                                ]
+                            },
+                            {
+                                "title": "Wird das positive Testergebnis bzw. das Warnen in meinem Kontakt-Tagebuch vermerkt?",
+                                "anchor": "warn_without_tan_contact_journal",
+                                "active": false,
+                                "textblock": [
+                                    "Ja, wenn Sie einen positiven Testbefund (auch ohne TAN) in der Corona-Warn-App registrieren, wird dieser Test wie bisher auch im Kontakt-Tagebuch eingetragen. Das geschieht unabhängig von der Art des Tests.",
+                                    "Wenn Sie manuell über die neue Funktion \"Tests erfassen und andere warnen\" andere CWA-Anwender*innen direkt warnen, wird zwar kein Test in der App registriert, aber im Tagebuch dokumentiert, dass Sie Andere gewarnt haben.",

```

Comment:
This is somehow confusing, I think.
The first sentence is:
"Ja, wenn Sie einen positiven Testbefund (**auch ohne TAN**) in der Corona-Warn-App registrieren, wird dieser Test **wie bisher auch** im **Kontakt-Tagebuch eingetragen**. Das geschieht unabhängig von der Art des Tests."

after that the entry continues with the sentence:
"Wenn Sie manuell über die neue Funktion \"Tests erfassen und andere warnen\" andere CWA-Anwender*innen direkt warnen, wird zwar **kein Test in der App registriert**, aber im Tagebuch **dokumentiert**, dass Sie **Andere gewarnt** haben."

So to me, this sounds like that a positive test registered using the SRS feature is NOT documented in the contact journal with an entry "Positiver Test registriert" but only the entry "Warnung ausgelöst" is added. Is my understanding correct? If yes, I will come up with a suggestion to improve this.

Reply:
Your understanding is correct. Using the traditional method, you get an entry for the registered test and the warning, and when using the new feature, you only get an entry for the warning. We think that the current text is sufficient, but would still appreciate a suggestion. 

![MicrosoftTeams-image (71)](https://user-images.githubusercontent.com/88365935/212309453-b9b5a95e-db82-42e4-83a2-d6ac7c81bf51.png)
![MicrosoftTeams-image (72)](https://user-images.githubusercontent.com/88365935/212309525-fa5625fe-e1dd-4ba8-8b69-7acd95426a06.png)

Reply:
@larswmh 

I will provide a suggestion for a clearer text in a few hours!

Reply:
```suggestion
                                    "Haben Sie das Ergebnis eines PCR-Tests oder Schnelltests mit Hilfe eines QR-Codes oder einer TAN in die Corona-Warn-App eingepflegt, erscheint im Tagebuch ein entsprechender Eintrag.",
                                    "Wenn Sie manuell über die Funktion \"Tests erfassen und andere warnen\" andere CWA-Anwender*innen direkt warnen, wird kein Test in der App registriert, es wird also auch kein entsprechender Eintrag im Tagebuch erstellt.",
                                   "Wenn Sie über die App gewarnt haben, sehen Sie unabhängig von der Art des Tests und wie Sie ihn registriert haben, im Tagebuch das Datum der Warnung.",
```

Reply:
@Ein-Tim thanks for your suggestion! I've changed "Tagebuch" to "Kontakt-Tagebuch" and fixed the indentation on the third line, along with the translation to the English FAQ in this commit: https://github.com/corona-warn-app/cwa-website/pull/3322/commits/0421f09d691ced522fa4a0229a49a08342f15579

Apart from that, we've got an answer on one of your questions [here](https://github.com/corona-warn-app/cwa-website/pull/3322#discussion_r1069218577). I'm not sure if you've seen it, but to be safe I'll let you know again